source/_posts/angular-analytics.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Google analytics with Tag manager and Angular
+date: 2020-10-25 13:14:41
+author: Abhishek Rana
+category:
+  - ["Web"]
+tags:
+  - Google Analytics
```

Comment:
use keyword in content also which is placed in tag section right now, some of them is not used any where in the content.